cuegui/cuegui/Utils.py

```diff
@@ -872,3 +873,21 @@ def byteConversion(amount, btype):
     for _ in range(n):
         _bytes *= 1024
     return _bytes
+
+
+def isPermissible(jobObject):
```

Comment:
Suggestion:

Use `snake_case` for Python variables (PEP = Python Enhancement Proposals):

jobObject -> job_object

hasPermissions -> has_permissions

currentUser -> current_user